gpiozero/boards.py

```diff
@@ -795,10 +795,100 @@ def __init__(self, location=None, pwm=False, initial_value=False):
                                         pwm=pwm, initial_value=initial_value)
 
 
+class StatusZero(LEDBoard):
+    """
+    Extends :class:`LEDBoard` for The Pi Hut's `STATUS Zero`_: a Pi Zero sized
+    add-on board with three sets of red/green LEDs to provide a status
+    indicator.
+
+    The following example designates the first strip the label "wifi" and the
+    second "raining", and turns them green and red respectfully::
+
+        from gpiozero import StatusZero
+
+        status = StatusZero('wifi', 'raining')
+        status.wifi.green.on()
+        status.raining.red.on()
+
+    :param str \*labels:
+        Specify the names of the labels you wish to designate the strips to.
+        You can list up to three labels. If no labels are given, three strips
+        will be initialised with names 'one', 'two', and 'three'. If some, but
+        not all strips are given labels, any remaining strips will not be
+        initialised.
+
+    .. _STATUS Zero: https://thepihut.com/statuszero
+    """
+    def __init__(self, *labels, **kwargs):
+        pins = (
+            (4, 17),
+            (27, 22),
+            (10, 9),
+        )
+        if len(labels) == 0:
+            labels = ['one', 'two', 'three'][:len(pins)]
```

Comment:
No - it came from the previous implementation which had a base class, so this was necessary.